cmd/appgw-ingress/main.go

```diff
@@ -79,6 +80,12 @@ func main() {
 
 	env.Consolidate(cpConfig)
 
+	// adjust ingress class value if overridden by environment variable
+	if env.IngressClass != "" {
```

Comment:
Maybe we can do the assignment in utils.go

Reply:
Thanks. I will look into this.

Reply:
@3quanfeng are you thinking something along the lines of:

`utils.SetIngressClass(env)`

Reply:
> @3quanfeng are you thinking something along the lines of:
> utils.SetIngressClass(env)

Yeah, something like getNamespacesToWatch(env.WatchNamespace), plus "pkg/annotations" is already imported in utils